ganache.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env sh
+npx ganache-cli "$@"
```

Comment:
If we have this, we need also

`cmd: ./ganache.sh` in brownie-config.yaml
and ganache in devDependencies in package.json

It's needed to use local project dependent ganache installation.